packages/camp/package.py

```diff
@@ -107,20 +107,28 @@ def blt_link_helpers(options, spec, spec_compiler):
             "/usr/tce/packages/gcc/gcc-4.9.3/lib64;/usr/tce/packages/gcc/gcc-4.9.3/gnu/lib64/gcc/powerpc64le-unknown-linux-gnu/4.9.3;/usr/tce/packages/gcc/gcc-4.9.3/gnu/lib64;/usr/tce/packages/gcc/gcc-4.9.3/lib64/gcc/x86_64-unknown-linux-gnu/4.9.3"))
 
     compilers_using_toolchain = ["pgc++", "xlc++", "xlC_r", "icpc", "clang++", "icpx"]
-    if any(compiler in spec_compiler.cxx for compiler in compilers_using_toolchain):
+    if any(tc_comp in compiler.cxx for tc_comp in compilers_using_toolchain):
```

Comment:
`spec_compiler` was a bad (confusing) name, since the parameter value is typically `self.compiler` and not `self.spec.compiler`.

Reply:
Yes, this is more clear.